eng/scripts/Create-APIView.ps1

```diff
@@ -86,6 +86,30 @@ function Get-ResourceProviderFromReadMePath {
     return $null
 }
 
+function Get-ImpactedTypespecProjects {
+    param (
+        [Parameter(Mandatory = $true)]
+        [string]$TypeSpecFile
+    )
+    $projectRootPaths = [System.Collections.Generic.HashSet[string]]::new()
+    $filePathParts = $TypeSpecFile.split([IO.Path]::DirectorySeparatorChar)
+    $typeSpecProjectBaseDirectory = $filePathParts[0..($($filePathParts.Length)-2)] -join [IO.Path]::DirectorySeparatorChar
```

Comment:
This will break if TypeSpec file is added in one more sub directory. What if changed file is tspconfig.yaml and and main.tsp.

For e.g

./specification/<service>/<project>/main.tsp
./specification/<service>/<project>/tspconfig.yaml
./specification/<service>/<project>/subfolder/subfolder1/a.tsp

In case of first two files in changes files, this code will find all projects in the specification. right? And in case of last e.g. it won't even detect the project because it will look for tspconfig.yaml in `./specification/<service>/<project>/subfolder/`

Can we use bottom-up search to start looking for tspconfig.yaml and main.tsp from changed files directory and look upwards until '/specification/service' and identify unique list of projects?